server/router.js

```diff
@@ -75,6 +75,11 @@ module.exports = function(database, config) {
 
     router.post('/users/:user/lists', listController.createList.bind(listController));
 
+    router.post('/lists', function(request, response){
```

Comment:
This rly should be a GET instead of POST, but it's fine for now.

Reply:
yeah i know, i was just lazy and copy/pasted the controller boilerplate and then didn't want to look up what the property was that stored url params

i guess, shockingly, it is `params`

Reply:
⚡ 💡